securesystemslib/signer/_crypto_signer.py

```diff
@@ -204,6 +212,13 @@ def from_priv_key_uri(
         expected (and tested) encoding/format is PEM/PKCS8. Other formats may
         but are not guaranteed to work.
 
+        URI has the format "file2:<PATH>", where PATH is a filesystem path to the
+        private key file. If CRYPTO_SIGNER_PATH_PREFIX environment variable
+        is set, the private key will be read from
+        ``CRYPTO_SIGNER_PATH_PREFIX + <SEPARATOR> + PATH``. The purpose of this
+        is to allow PATH to only encode an identifier (e.g. filename) while allowing
+        the signing system to store the private keys where ever it wants at runtime.
```

Comment:
spelling nit2:
```suggestion
        the signing system to store the private keys wherever it wants at runtime.
```

Regardless, do you think it's necessary to have the same block here and in the class docstring?